docs/developer-guide/ksqldb-clients/java-client.md

```diff
@@ -643,6 +644,30 @@ for (QueryInfo query : queries) {
 }
 ```
 
+See the [API reference](https://docs.ksqldb.io/en/latest/developer-guide/ksqldb-clients/java-client/api/io/confluent/ksql/api/client/Client.html)
+for more information.  
+
+Describe specific streams and tables<a name="describe-source"></a>
+------------------------------------------------------------------------
```

Comment:
Maybe use `##` instead of `--------`, just for consistency with the ### headings.

Reply:
Is it possible to provide custom anchors using the `##` notation? The reason I used `-------` was in order to provide short anchors for the long titles (not just this one but throughout the entire document).

Reply:
Good question, I have't tried that!

Reply:
Couldn't find a way to make it work, planning to merge as is.